docs/beginner-checks.md

```diff
@@ -0,0 +1,74 @@
+# Getting Started with Scorecard Checks for Supply Chain Security
+
+Choosing which Scorecard checks to get started with as a project maintainer can be overwhelming. This page walks through some of the most important checks to start with for project improvement, focusing on the ones that give you the biggest payoff versus effort. They're broken down into three categories based on stages of the development process: setting up your project, accepting contributions from others, and packaging the project to release to the world.
```

Comment:
might be nice to make this list act as a mini table of contents, and link to the three sections?

Reply:
The links will need fixed slightly, for example for the first one it's looking for:
`#1-setting-up-your-project`. (you can see this by previewing the file and clicking on the "link" button for the section)